exalearn-ui/src/app/pages/create-page/create-page.component.scss

```diff
@@ -3,9 +3,14 @@
 .create {
 	height: 100%;
 	max-height: 100%;
+	display: flex;
+	flex-direction: row;
```

Comment:
It looks brilliant.  But one small remark :trollface: 
row - default flex-direction value